### ISSUE TITLE
Error: stream_copy_to_stream(): Write of 4096 bytes failed with errno=32 Broken pipe

### ISSUE THREAD

**Issue by paulcalabro** [pulsejet/memories#60]
I'm seeing tons of identical error messages in my logs when performing indexing:

```
Error: stream_copy_to_stream(): Write of 4096 bytes failed with errno=32 Broken pipe at /var/www/html/custom_apps/memories/lib/Exif.php#219
```

**Comment by paulcalabro** [comment]
It looks like I'm hitting this 20MB limit:

https://github.com/pulsejet/memories/blob/373d37ecf106a595801fad63988f6218d3886849/lib/Exif.php#L216-L219

**Comment by pulsejet** [comment]
That shouldn't be the problem. What version of exiftool is this? Also what platform / PHP version etc?

**Comment by ssc85** [comment]
I am also seeing such kind of errors using Nextcloud 25 on Truenas / Freebsd. The log file is full of the below shown errors:

`
Error: stream_copy_to_stream(): Write of 8192 bytes failed with errno=32 Broken pipe at /usr/local/www/nextcloud/apps/memories/lib/Exif.php#158
`

I am using the built-in exiftool as well as the one available within Freebsd 13.1-RC6  - 12.49. The used php version is PHP 8.0.24 (fpm-fcgi)

**Comment by pulsejet** [comment]
Please try with v4.5.2
